docs/input/docs/usage/examples.md

```diff
@@ -0,0 +1,80 @@
+---
+Order: 20
+Title: Examples
+Description: Examples of usages of BBT.Maybe
+---
+
+## Usage of factory methods
+
+For the creation of maybe values factory methods are provided. Methods with arguments for reference are available (Some method). For the none case parameterless methods can be used (None method).
+For nullable value types analogous methods postfixed with 'Struct' are provided.
+
+```csharp
+    // Creation of a maybe for a declared reference type which could be instantiated or not.
```

Comment:
Remove indentation as this indentates the code block's content.

Reply:
Done.